docs/competition/event/incident-management.md

```diff
@@ -80,7 +80,7 @@ If the match is stopped immediately, if safe to do so, the Marshals should hit t
 
 In either scenario; the Badgers should ask all competitors to leave the staging area. Robots to remain where they are, power cut offs to be hit.
 
-## Resuming/cancelling the competition
+#### Resuming/cancelling the competition
```

Comment:
Not sure why this should be a subsection of stopping matches?
```suggestion
### Resuming/cancelling the competition
```

Reply:
I think because to resume the competition, matches have to have been stopped, but I agree this probably reads better.